src/index.ts

```diff
@@ -0,0 +1,75 @@
+import { unicodeLength, timeToInt } from './utils';
+
+const REGEX_EMAIL = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+const REGEX_INT = /^\d+$/;
+const REGEX_FLOAT = /^\d+(\.\d+)?$/;
+const REGEX_PHONE = /^(?=.{2,50}$)[/\-.()+\s\d–]*\d{2,}[/\-.()+\s\d–]*$/;
+const REGEX_URL = /^(?:https?:\/\/)?((?:[^\u0000-\u007F]|[\w-])+\.(?:(?:[^\u0000-\u007F]|[\w-]){2,}\.)*(?:[^\u0000-\u007F]|[a-zA-Z])+)(?::\d+)?(\/(?:(?:[^\u0000-\u007F]|[\w/.,'"@+=!:;=%()-])+)?(?:\.(?:[^\u0000-\u007F]|[\w-])+)?)?((?:\?|#)\S+)?$/;
+
+const REGEX_LOWERCASE = /[a-z]+/;
+const REGEX_UPPERCASE = /[A-Z]+/;
+const REGEX_NUMBERS = /\d+/;
+const REGEX_SPECIAL = /[^A-Za-z\d]+/;
+const REGEX_TIME = /^\d{2}:\d{2}$/;
+
+export const isEmpty = (value?: any): boolean => typeof value === 'undefined' || value === null;
```

Comment:
maybe rather go with `unknown` in most of those cases? not too sure myself. just know that we should prefer `unknown` since it's more defensive then `any`